tests/tensor_search/integ_tests/test_search_unstructured.py

```diff
@@ -1441,3 +1442,116 @@ def run():
                       vespa_query_kwargs["yql"])
         self.assertIn("embedding_query", vespa_query_kwargs["query_features"])
         self.assertNotIn("marqo__query_embedding", vespa_query_kwargs["query_features"])
+
+    def test_special_characters_in_map_score_modifiers(self):
+         special_characters = [
```

Comment:
nit. the special chars are duplicated in three tests, can we extract it?

Reply:
Done